filesystem/src/main/java/org/frankframework/filesystem/ForEachAttachmentPipe.java

```diff
@@ -179,9 +180,29 @@ public void setExcludeProperties(String excludePropertiesList) {
 		if (excludeProperties==null) {
 			excludeProperties=new LinkedHashSet<>();
 		}
-		Misc.addItemsToList(excludeProperties,excludePropertiesList,"properties not to list",false);
+		addItemsToList(excludeProperties, excludePropertiesList, "properties not to list");
 	}
 	public Set<String> getExcludePropertiesSet() {
 		return excludeProperties;
 	}
+
+	/**
+	 * Adds items on a string, added by comma separator (ex: "1,2,3"), into a list.
+	 * @param collectionDescription description of the list
+	 */
+	void addItemsToList(Collection<String> collection, String list, String collectionDescription) {
```

Comment:
To match the first line of the Javadoc, I'd change the names of the vars to `list` for the first one, and rename the 2nd var from `list` to `input` or at least something that not mixes with the output list.